foundry.toml

```diff
@@ -5,6 +5,8 @@ libs = ["lib"]
 fs_permissions = [{ access = "read-write", path = "./" }]
 solc_version = "0.8.25"
 build_info = true
+ffi = true
```

Comment:
why is ffi needed?

Reply:
As defined [here](https://docs.openzeppelin.com/upgrades-plugins/1.x/foundry-upgrades#before_running)